formbricks/formbricks
# chore: move group components in app folder

## What does this PR do?



Implements new folder structure
![image](https://github.com/formbricks/formbricks/assets/31730715/54fa72cc-af54-4e34-9114-85087c629ab4)




## Type of change




## How should this be tested?



- Run pnpm build
- Perform a sanity check to make sure everything works fine
- Look at the folder structure

## Checklist



### Required


### Appreciated
